### ISSUE TITLE
上架google商店提示StrictMode policy violation: android.os.strictmode.NonSdkApiUsedViolation: Landroid/view/View;->getAccessibilityDelegate()Landroid/view/View$AccessibilityDelegate;

### ISSUE THREAD

**Issue by Gloud-Zhao** [tencentyun/TIMSDK#167]
![image](https://user-images.githubusercontent.com/13994354/59394604-dba17f00-8db2-11e9-81b5-af26aa8ad029.png)
上加后有禁告
StrictMode policy violation: android.os.strictmode.NonSdkApiUsedViolation: Landroid/view/View;->getAccessibilityDelegate()Landroid/view/View$AccessibilityDelegate;

**Comment by Gloud-Zhao** [comment]
![image](https://user-images.githubusercontent.com/13994354/59394682-2f13cd00-8db3-11e9-8619-2a93a27ffbcb.png)

**Comment by whalehe** [comment]
1、第1个图com.tendcloud.tenddata引起，不是TIMSDK中引入，建议去TalkingData Analytics咨询此问题
2、第2个图com.tencent.tbs，不是TIMSDK中引入，建议去tbs团队咨询此问题：https://x5.tencent.com/tbs/guide/sdkInit.html
